### DISCUSSION TITLE
Any change with support for ILI9225 2.2" display

### DISCUSSION THREAD

**Discussion by griffinmt** [Bodmer/TFT_eSPI#2473] [Compatible displays and setup files]
I just got a non-touch version of this screen, but realized it cannot be selected in the Setup files. How do I go about using it with your library.

TIA.
Martyn

**Comment by Bodmer** [comment]
The ILI9225 is supported with SPI connections. Either edit User_Setup.h to suit your processor, pins etc and use:
#define ILI9225_DRIVER

Or create a  new setup file and load it by editting User_Setup_Select.h to load that file. Setup 42 might be a good starting point if you are using an ESP32.

**Comment by griffinmt** [comment]
Thanks for the response. I actually sent it back to amazon as it wasnt what
i originally ordered.
